pkg/chains/storage/grafeas/grafeas_test.go

```diff
@@ -0,0 +1,382 @@
+					Attestation: &attestationpb.Details{
+						Attestation: &attestationpb.Attestation{
+							Signature: &attestationpb.Attestation_GenericSignedAttestation{
+								GenericSignedAttestation: &attestationpb.GenericSignedAttestation{
+									// ContentType: for taskrun, this will be GenericSignedAttestation_CONTENT_TYPE_UNSPECIFIED, so this will not show up
+									SerializedPayload: []byte("taskrun payload"),
+									Signatures: []*commonpb.Signature{
+										{
+											Signature: []byte("taskrun signature"),
+											// PublicKeyId: we're only using KMS for signing which is the one we currently set its reference in attestation
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+				Envelope: &commonpb.Envelope{
+					Payload:     []byte("taskrun payload"),
+					PayloadType: "in-toto attestations containing a slsa.dev/provenance predicate",
+					Signatures: []*commonpb.EnvelopeSignature{
+						{
+							Sig: []byte("taskrun signature"),
+							// PublicKeyId: we're only using KMS for signing which is the one we currently support for storing its reference in attestation
+						},
+					},
+				},
+			},
+			// occurrence for OCI image
+			{
+				Name:     "gcr.io/test/kaniko-chains1@sha256:cfe4f0bf41c80609214f9b8ec0408b1afb28b3ced343b944aaa05d47caba3e00",
+				Resource: &pb.Resource{Uri: "gcr.io/test/kaniko-chains1@sha256:cfe4f0bf41c80609214f9b8ec0408b1afb28b3ced343b944aaa05d47caba3e00"},
+				NoteName: "projects/test-project/notes/test-note",
+				Details: &pb.Occurrence_Attestation{
+					Attestation: &attestationpb.Details{
+						Attestation: &attestationpb.Attestation{
+							Signature: &attestationpb.Attestation_GenericSignedAttestation{
+								GenericSignedAttestation: &attestationpb.GenericSignedAttestation{
+									ContentType:       attestationpb.GenericSignedAttestation_SIMPLE_SIGNING_JSON,
+									SerializedPayload: []byte("oci payload"),
+									Signatures: []*commonpb.Signature{
+										{
+											Signature: []byte("oci signature"),
+											// PublicKeyId: we're only using KMS for signing which is the one we currently support for storing its reference in attestation
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+				Envelope: &commonpb.Envelope{
+					Payload:     []byte("oci payload"),
+					PayloadType: "in-toto attestations containing a slsa.dev/provenance predicate",
+					Signatures: []*commonpb.EnvelopeSignature{
+						{
+							Sig: []byte("oci signature"),
+							// PublicKeyId: we're only using KMS for signing which is the one we currently support for storing its reference in attestation
+						},
+					},
+				},
+			},
+		},
+	}
+	got, err := client.ListOccurrences(ctx,
+		&pb.ListOccurrencesRequest{
+			// This is just a placeholder.
+			// ProjectID doesn't matter here because we assume there is only one project in the mocked server.
+			Parent: fmt.Sprintf("project/PLACEHOLDER"),
```

Comment:
Should we just omit this then?

Reply:
the tests still pass if we omit this. just though it might be useful to put here to show how a `ListOccurrencesRequest` is created. what do you think about it?